tests/utils/test_raster.py

```diff
@@ -241,3 +273,40 @@ def test_sum_resample_file(self, resample_xarr_fpath: str) -> None:
         # assert correct resampling values (summing consitiuent grids)
         expected_result = np.array([[[14, 22], [46, 54]]])
         assert np.array_equal(expected_result, xds_out.to_numpy())
+
+    @pytest.mark.parametrize(
```

Comment:
I would also suggest to possibly include a test that looks for a directory that doesn't exist to accompany the tests that looks for a file that does not exist

Reply:
I'm not sure this test case is a valid one. The input to this function is a file path, and the existence of the file is what's being check. So, if the directory doesn't exist then input file can't exist either. Nonetheless, it's a very quick test to do so I've added it in commit f71625b. Let me know what you think. I'm happy to resolve this conversation if you are too.

Reply:
I agree that it may not be needed. Never the less it's only a small test and a few lines so it wouldn't hurt to have. I'm happy to merge this PR with dev once the runners have passed. Thanks for the quick changes @ethan-moss !